tests/testColor.py

```diff
@@ -308,11 +275,6 @@ def defineFilterProperty(self, name, lambdaEff, force=False):
     def testListFilters(self):
         self.assertEqual(afwImage.Filter_getNames(), self.filters)
 
-    def testCtor(self):
```

Comment:
There are no asserts and there are many other tests that make exactly that function call. It's not clear what the failure condition for this test would be, anyway.